### PR TITLE
Lab 2

### PR DESCRIPTION
**Pull request by molexted**
For relevant library versions see #1.

### CODE REVIEW

**Comment by keirashea** [comment]
Don't forget to complete Problem 1 parts (C) and (D) and Problem 4. You should also label the results you are printing in your code in part 2a and 3a. For your plot in 2a you should change the size of the figure using something like fig, ax = plt.subplots(1, 4, figsize=(25, 5)) because it's really hard to see the first couple of distributions since they are so compressed.

**Comment by molexted** [comment]
@oliviatessa PDF now included as of f22e8e8.

**Comment by oliviatessa** [comment]
Hi Alex, 

Here are my comments for your lab 2:

P.1
- Could you be more specific than "Arbitrary value for this problem is 7?" Maybe try introducing the problem and telling the reader what 7 means in the context of the problem? 
- Actually, you can use the CLT to approximate a Poisson with a Gaussian for a much smaller number
- In your last plot, it might be useful to plot a Gaussian alongside to illustrate how the Poisson converges to Gaussian with convolutions. Especially helpful to look at this on a log scale 

P.2 
- I'm not sure what you are talking about when you say "There are a good number of fudge factors in the code..." Is this paragraph pertinent for understanding the statistical concepts explored in this lab? 

P.3, P.4, and generally
- Your writing can be hard to follow at times. Clarity in writing is extremely important in scientific documents and also one of the most important skills we expect you to develop in this class. Try reading your writing out loud to yourself before you submit to make sure the sentences make sense. 

In your report: 
"As seen above, the trials penalty is about four and a half, from 15 to 19.56. This means that the distribution's integral at small values is does not have a hugely larger value despite the increase in the number of samples."

The above sentence could be rewritten to be more clear, grammatically correct, and easier to understand. 

Best, 
Olivia

**Comment by molexted** [comment]
Opening to merge.
